File: fix_users.js
const fs = require('fs');

const path = 'lib/models/Users.dart';
let content = fs.readFileSync(path, 'utf8');

const lines = content.split('\n');
const newLines = [];
let skip = false;

for (let line of lines) {
    if (line.includes('modelSchemaDefinition.addField(') && (line.includes('Users.CREATED_AT') || line.includes('Users.LAST_SEEN'))) {
        skip = true;
        continue;
    }
    if (skip) {
        if (line.includes(');')) {
            skip = false;
        }
        continue;
    }
    
    if (line.includes('created_at') || line.includes('last_seen') || line.includes('CREATED_AT') || line.includes('LAST_SEEN')) {
        continue;
    }
    
    newLines.push(line);
}

fs.writeFileSync(path, newLines.join('\n'), 'utf8');
console.log('Fixed Users.dart');

File: fix_users.py
import re

with open(r"d:\Cochin United\Cochin United\CUC Main Version\lib\models\Users.dart", "r", encoding="utf-8") as f:
    content = f.read()

lines = content.split('\n')
new_lines = []
skip = False
for line in lines:
    if "modelSchemaDefinition.addField(" in line and ("Users.CREATED_AT" in line or "Users.LAST_SEEN" in line):
        skip = True
        continue
    if skip:
        if ");" in line:
            skip = False
        continue
    
    if "created_at" in line or "last_seen" in line or "CREATED_AT" in line or "LAST_SEEN" in line:
        continue
        
    new_lines.append(line)

with open(r"d:\Cochin United\Cochin United\CUC Main Version\lib\models\Users.dart", "w", encoding="utf-8") as f:
    f.write('\n'.join(new_lines))

print("Fixed Users.dart")
